profiler/src/ProfilerEngine/Datadog.Profiler.Native/LibddprofExporter.cpp

```diff
@@ -249,15 +249,22 @@ ddog_Endpoint LibddprofExporter::CreateEndpoint(IConfiguration* configuration)
         // Agent mode
 
 #if _WINDOWS
-        bool useDefaultDomainSocket = false;
+        std::string socketPath;
+        const std::string& namePipeName = configuration->GetNamedPipeName();
+        if (!namePipeName.empty())
+        {
+            socketPath = R"(\\.\pipe\)" + namePipeName;
+        }
+        std::string agentUrl = "windows:" + socketPath;
 #else
-        std::error_code ec; // fs::exists might throw if no error_code parameter is provided
-        bool useDefaultDomainSocket = fs::exists("/var/run/datadog/apm.socket", ec);
+        std::string socketPath = "/var/run/datadog/apm.socket";
+        std::string agentUrl = "unix://" + socketPath;
```

Comment:
Windows supports UDS too (we explicitly test that in the tracer)

Reply:
Today libdatadog does not support UDS on windows. We can leave it for another review (once it's available in libdatadog)